build/scripts/version-update/after-release/after-release.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("Please provide the initial version as a command-line argument")
+		return
+	}
+
+	initialVersion := os.Args[1]
+	files := []string{
+		"install/helm/agones/Chart.yaml",
+		"install/yaml/install.yaml",
+		"install/helm/agones/values.yaml",
+		"sdks/nodejs/package.json",
+		"sdks/nodejs/package-lock.json",
+		"sdks/unity/package.json",
+		"sdks/csharp/sdk/AgonesSDK.nuspec",
+		"sdks/csharp/sdk/csharp-sdk.csproj",
+	}
+
+	for _, filename := range files {
+		err := UpdateVersionInFile(filename, initialVersion)
+		if err != nil {
+			fmt.Printf("Error updating file %s: %s\n", filename, err.Error())
```

Comment:
maybe a `log.Fatalf` instead of just a Printf?

Reply:
used log instead fmt